src/Persistence/Fake/FakeResetPasswordInternalRepository.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace SymfonyCasts\Bundle\ResetPassword\Persistence\Fake;
+
+use SymfonyCasts\Bundle\ResetPassword\Exception\FakeRepositoryException;
+use SymfonyCasts\Bundle\ResetPassword\Model\ResetPasswordRequestInterface;
+use SymfonyCasts\Bundle\ResetPassword\Persistence\ResetPasswordRequestRepositoryInterface;
+
+/**
+ * Class is only used as a placeholder and it's signature should be replaced in reset_password.yaml.
```

Comment:
its "signature" should be replaced? It's more that the config key should be changed to something else?